.mci.yml

```diff
@@ -293,6 +295,9 @@ functions:
                       ${test_params} ./src/mongocxx/test/test_command_monitoring_specs
                       ${test_params} ./src/mongocxx/test/test_instance
                       ${test_params} ./src/mongocxx/test/test_transactions_specs
+                      ${test_params} ./src/mongocxx/test/test_logging
+                      ${test_params} ./src/mongocxx/test/test_retryable_reads_specs
```

Comment:
The danger of having many test runners is that easy miss some, either on evergreen or locally. When we have time, I'd like to simplify this to one test runner (or have one script that runs them all), and remove the `*_TESTS_PATH` environment variables.

Reply:
+1 for cleaning up the test runners!